index.js

```diff
@@ -780,10 +780,10 @@ sanitizeHtml.defaults = {
   disallowedTagsMode: 'discard',
   allowedAttributes: {
     a: [ 'href', 'name', 'target' ],
-    // We don't currently allow img itself by default, but this
+    // We don't currently allow img itself by default, but these
     // would make sense if we did. You could add srcset here,
     // and if you do the URL is checked for safety
-    img: [ 'src' ]
+    img: [ 'src', 'alt', 'title', 'width', 'height', 'loading' ]
```

Comment:
This is reasonable — how about adding `srcset` while you're at it?

Reply:
Also be sure to update the changelog.

Reply:
Okay